### PR TITLE
Add an .editorconfig control file for vscode

### PR DESCRIPTION
**Pull request by dellis1972**
Context http://editorconfig.org/

When editing files in vscode it does tend
to mess up the formatting. This file will
let vscode know what our team's formatting
rules are.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+indent_style = tab
+indent_size = 4
+
+[*.{props,targets}]
+indent_style = space
+indent_size = 2
+
+[*.{csproj}]
+indent_style = space
+indent_size = 2
```
Comment (jonathanpeppers) [inline .editorconfig]: I think you can add more file extensions:
```
[*.{xml,axml,xaml,targets,proj,csproj,props}]
indent_style = space
indent_size = 2
```

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,32 @@
+; EditorConfig to support per-solution formatting.
+; Use the EditorConfig VS add-in to make this work.
+; http://editorconfig.org/
+
+[*.{cs}]
```
Comment (jonpryor) [inline .editorconfig]: Why is `*.{cs}` in two separate sections, here and line 18?

**Comment by kzu** [comment]
Full list of options: https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference?view=vs-2017

**Comment by jonpryor** [comment]
See also: https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference?view=vs-2017

**Comment by jpobst** [comment]
I am closing this out in favor of a more complete `.editorconfig` I did here: https://github.com/xamarin/xamarin-android/pull/3206.

**Pull request closed without merging**